src/js/Accordion.js

```diff
@@ -127,6 +129,7 @@ export default  class Accordion {
         }
 
         accordionHeader.classList.add("is-opened");
+        accordionHeader.parentNode.classList.add("is-opened");
```

Comment:
Provjeri da li "accordionHeader.parentNode"  ima "js-accordion-single" klasu (i na liniji 121, 88). Bez provjere može biti situacija da će parent od headera biti "js-accordion" (index.html linija 106), a to ne želimo jer može druge stvari strgati.

Reply:
fixed